app/helpers/react_components/mentoring/discussion.rb

```diff
@@ -19,6 +19,7 @@ def to_s
             },
             track: {
               title: track.title,
+              highlighter_language: track.highlighter_language,
```

Comment:
Could we scope this directly to the library please? So `highlightjs_language`.

The reason for this is that if we change libs or use both monaco and highlightjs within this component, we'll need the two seperate values and this will allow us to do that.